po/de.po

```diff
@@ -1380,12 +1380,12 @@ msgstr "Belichtung"
 #: ../build/lib/darktable/plugins/introspection_retouch.c:238
 #: ../build/lib/darktable/plugins/introspection_retouch.c:379
 msgid "blur type"
-msgstr "Weichzeichnungstyp"
+msgstr "Weichzeichner-Filter"
 
 #: ../build/lib/darktable/plugins/introspection_retouch.c:244
 #: ../build/lib/darktable/plugins/introspection_retouch.c:383
 msgid "blur radius"
-msgstr "Weichzeichnungsradius"
+msgstr "Weichzeichnersradius"
```

Comment:
```suggestion
msgstr "Weichzeichnungsfilter Radius"
```

Reply:
I think "Weichzeichnungsfilter Radius" and "Weichzeichner Radius" are not proper German translations of the English open (spaces) compound noun "blur radius". A compound noun should be used for the German translation too. "Weichzeichner Radius" are just two German words. 

The proposed term "Weichzeichnersradius" contains a superficial s, should be "Weichzeichnerradius".  One might argue that the long German compound nouns are hard to read in the UI. Therefore, "Radius des Weichzeichners" might be considered as an alternative (in combination with "Typ des Weichzeichners" for "type for the blur algorithm", see suggestion by @da-phil below.).

Reply:
Thank you.
Also, do you have any other opinions on the "filmisch" vs. "filmic" vs. (maybe, or maybe not) "Filmkurve" discussion at the beginning of this discussion?

Reply:
"Filmkurve" sounds a little bit artificial to me. Through, it still much better as all alternatives that I can think of. I particularly dislike the verbatim translation "filmisch" of the English term "filmic". Both terms commonly refer to characteristics of the *artistic genres* film and photography, see 
* https://dictionary.cambridge.org/de/worterbuch/englisch/filmic
* https://www.collinsdictionary.com/dictionary/english/filmic
* https://www.duden.de/rechtschreibung/filmisch

I can hardly imagine that a German native speaker associates "filmisch" with *technical* aspects of analog film. Therefore, the name "filmisch" is not very useful for explaining what the filmic module is about (emulating the response curve of analog film).

I thought about names as "Filmemulation" ("film emulation") or "Tonkurve" (tone curve). But these terms have their own drawbacks. "Filmemulation" usually refers to the emulation of *specific* analog films, for "Tonkurve" I would expect a general curve tool as we have it in dt as well with the Lab and RGB curves.

Thus, I would vote for "Filmkurve".

Reply:
I'd also go with the proposal of @rabauke "Weichzeichnerradius" for this correction.